jujugui/static/gui/src/app/app.js

```diff
@@ -2994,6 +2998,15 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Sets the page title.
+      @method setPageTitle
```

Comment:
You don't actually need to provide the `@method` but you do have to add the `@param`